packages/suite/src/views/settings/debug/CoinjoinApi.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { COINJOIN_SERVERS } from '@suite/services/coinjoin/config';
+import { ActionColumn, ActionSelect, SectionItem, TextColumn } from '@suite-components/Settings';
+import * as suiteActions from '@suite-actions/suiteActions';
+import { useSelector, useActions } from '@suite-hooks';
+import type { CoinjoinServerEnvironment } from '@suite-common/wallet-types';
+// import { useAnchor } from '@suite-hooks/useAnchor';
+// import { SettingsAnchor } from '@suite-constants/anchors';
+
+const StyledActionSelect = styled(ActionSelect)`
+    min-width: 256px;
+`;
+
+export const CoinjoinApi = () => {
+    const { setDebugMode } = useActions({
+        setDebugMode: suiteActions.setDebugMode,
+    });
+    const { debug } = useSelector(state => ({
+        debug: state.suite.settings.debug,
+    }));
+    // const { anchorRef, shouldHighlight } = useAnchor(SettingsAnchor.OAuthApi);
```

Comment:
is it worth to have an anchor for that?

Reply:
nice to have, I'd keep things simple for now.

Reply:
I don't think it's worth to have it.